rust/datafusion/tests/sql.rs

```diff
@@ -1496,6 +1496,25 @@ async fn csv_query_sum_cast() {
     execute(&mut ctx, sql).await;
 }
 
+#[tokio::test]
+async fn query_where_neg_num() -> Result<()> {
+    let mut ctx = ExecutionContext::new();
+    register_aggregate_csv_by_sql(&mut ctx).await;
+
+    // Negative numbers do not parse correctly as of Arrow 2.0.0
+    let sql = "select c7, c8 from aggregate_test_100 where c7 >= -2 and c7 < 10";
```

Comment:
It would be good to add a test to cover negative floating point numbers as well

Reply:
Done